### ISSUE TITLE
cpusets missing "0x" prefix when printed in hex mode

### ISSUE THREAD

**Issue by bgeltz** [karelzak/util-linux#744]
I noticed that the following patch removed the "0x" prefix from cpusets when they are printed in hex mode:
c82b12a0a23713e95c536d23bfc6ac2deb6a844c

Specifically, the following lines are identical whether or not you're in hex mode now:
https://github.com/karelzak/util-linux/blob/master/sys-utils/lscpu.c#L1689-L1695

Was the removal of "0x" intentional?

**Comment by karelzak** [comment]
Not sure what do you mean with "identical", I see difference in the output:
```
$ ./lscpu --hex | grep "On-line"
On-line CPU(s) mask: ff
$ ./lscpu | grep "On-line"
On-line CPU(s) list: 0-7
```
The mask is in compatible format like for example kernel uses in /sys or taskset on output:
```
$ taskset -p $$
pid 11356's current affinity mask: ff
```
Anyway you're right that before commit c82b12a there was 0x prefix in lscpu output and it's regression, but I'm not sure if we want to change it again after two years. It seems better to keep it...

**Comment by bgeltz** [comment]
Agreed that the output is indeed different.  I had missed the difference between the call to cpumask_create() vs. cpulist_create.

Next:
> The mask is in compatible format like for example kernel uses in /sys or taskset on output:

I understand the desire to align the printing of cpu masks between tools for consistency.  I did notice that ```taskset``` is capable of handling masks with or without the leading "0x" for input:
```
$ taskset -p 0xff $$
pid 29063's current affinity mask: ff
pid 29063's new affinity mask: ff
```

If this is not going to be fixed, perhaps the man page should be updated:
```man
       -x, --hex
              Use hexadecimal masks for CPU sets (for example 0x3).  The default is to print the sets in list format
              (for example 0,1).
```

To me, it seems more accurate to have the "0x" prefix present with the output.  However I understand that the code has been this way for a long time.  I'm just now bumping into this issue because I've updated my OS to a version that has a more recent version of ```lscpu```.  This broke one of my parsers.
